aten/src/ATen/core/ScalarType.h

```diff
@@ -165,25 +165,29 @@ static inline ScalarType promoteTypes(ScalarType a, ScalarType b) {
   constexpr auto f2 = ScalarType::Half;
   constexpr auto f4 = ScalarType::Float;
   constexpr auto f8 = ScalarType::Double;
+  constexpr auto c4 = ScalarType::ComplexFloat;
```

Comment:
nit: Can you name this c8 instead? That'll be consistent with `torch.complex64` and `torch.complex128`. c16 is too big, but you could maybe try `cd` instead...

Reply:
I thought `8` means the precision, so we have `c4` for float precision complex and `c8` for double precision complex which is consistent to `f4` and `f8`...

Reply:
oh all right :)